src/averageAges.js

```diff
@@ -34,7 +44,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const peopleWoman = people.filter(person => person.sex === 'f');
+
+  const peopleWomanAges = (withChildren)
```

Comment:
```suggestion
  const womenAges = (withChildren)
```
we know that women are people so we can skip this